mike/themes/material/js/version-select.js

```diff
@@ -1,49 +1,46 @@
 window.addEventListener("DOMContentLoaded", function() {
-  // This is a bit hacky. Figure out the base URL from a known CSS file the
-  // template refers to...
-  var ex = new RegExp("/?assets/fonts/material-icons.css$");
-  var sheet = document.querySelector('link[href$="material-icons.css"]');
-
-  var ABS_BASE_URL = sheet.href.replace(ex, "");
-  var CURRENT_VERSION = ABS_BASE_URL.split("/").pop();
-
+  var VERSION = window.location.pathname.split('/')[1];
+  
   function makeSelect(options, selected) {
-    var select = document.createElement("select");
-    select.classList.add("form-control");
-
-    options.forEach(function(i) {
-      var option = new Option(i.text, i.value, undefined,
-                              i.value === selected);
-      select.add(option);
-    });
-
-    return select;
+      var select = document.createElement("select");
```

Comment:
Looks like the indentation here got changed to 4 spaces; this project uses 2 space indentation for non-Python stuff (Python follows PEP8).